index.ts

```diff
@@ -59,42 +64,47 @@ export async function run() {
 // Membrane programs can also export _fields_ that resolve to values.
 // Fields can also contain subfields instead of pointing to a value directly.
 // This program contains a `status` field that resolves to a string.
-// Try changing the string returned, hit save, and see your text in the Explorer next to `getting-started`.
+// Try changing the string returned, hit save, and see your text in the Explorer (sidebar) next to `getting-started`.
 
 export function status() {
   return "👋 start here!";
 }
 
-// Note: `status` is an implicit field that all Membrane programs have, but you can add any fields to a schema.
-// To learn more about actions and fields in Membrane, visit: https://docs.membrane.io/concepts/schema
+// 💡 Note: `status` is an implicit field that all Membrane programs have, but you can add any fields to a schema.
+// 💡 There's also nothing special about the name `run` for an action.
+// We'll cover editing names and types for a program's actions and fields (aka its schema) in step 8 of this tutorial.
+// 🔗 To learn more about actions and fields in Membrane, visit: https://docs.membrane.io/concepts/schema
 
 /**
  * ========================================================================================================================
  * STEP 3 | Membrane `sms` and `email`
  * ========================================================================================================================
  */
 
-// Membrane comes with several built-in utils that your programs can install, like `sms` and `email`.
-// Your workspace has `sms` and `email` installed, and this program includes them as connections.
-// We'll cover how to add new connections later.
+// Membrane comes with a few built-in programs that your programs can use, like `sms` and `email`.
+// This `getting-started` program already includes `sms` and `email` as connections (similar to npm dependencies).
```

Comment:
Instead of saying `similar to npm dependencies, what if we just linked to the docs on connections?

Reply:
Never mind, I see you have it linked below.

Reply:
I'm hoping people may pick up on the pattern of link-to-learn-more at the bottom of each step, but that's probably wishful thinking. Maybe I could also add links right-justified in each step header?

Reply:
I think the link to learn more pattern is fine